src/Stepper/index.js

```diff
@@ -0,0 +1,74 @@
+// @flow srict
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import Text from '../Text';
+import Icon from '../Icon';
+import StepperButton from './StepperButton';
+import StyleSheet from '../PlatformStyleSheet';
+import type { StylePropType } from '../PlatformStyleSheet/StyleTypes';
+
+type Props = {|
+  +onIncrement: () => void,
+  +onDecrement: () => void,
+  +number: number,
+  +showNumber?: boolean,
+  +numberStyle?: StylePropType,
+  +min?: number,
+  +max?: number,
+  +disableDecrement?: boolean,
+  +disableIncrement?: boolean,
+|};
+
+export default function Stepper({
+  onIncrement,
+  onDecrement,
+  number,
+  showNumber = true,
+  numberStyle,
+  min,
+  max,
+  disableDecrement: originalDisableDecrement,
+  disableIncrement: originalDisableIncrement,
+}: Props) {
+  const disableDecrement = (min && number <= min) || originalDisableDecrement;
```

Comment:
`min && number <= min` and `max && number >= max` are used in some places. Maybe it's worth to extract it to const?

Reply:
Good idea, I'll do that!